test/functional/deprecated_rpc.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2016 The Bitcoin Core developers
```

Comment:
micronit: 2017

Reply:
Will be fixed by the "bump copyright headers" script ;)

Reply:
I was pushing anyone so I fixed this